.travis.yml

```diff
@@ -10,10 +10,22 @@ install: true
 matrix:
   include:
   - stage: test
+    name: "Test on Linux"
```

Comment:
Shouldn't that require `os: linux` for 100% symmetry?

Reply:
Yes, good point. Added now. Thanks.